src/main/java/mods/railcraft/common/blocks/aesthetics/cube/BlockCube.java

```diff
@@ -176,8 +176,7 @@ public boolean shouldSideBeRendered(IBlockAccess world, int x, int y, int z, int
 
     @Override
     public boolean canCreatureSpawn(EnumCreatureType type, IBlockAccess world, int i, int j, int k) {
-        int meta = world.getBlockMetadata(i, j, k);
-        return EnumCube.fromOrdinal(meta).getBlockDef().canCreatureSpawn(type, world, i, j, k);
+        return !RailcraftConfig.cannotMobsSpawnOnDecorBlocks;
```

Comment:
Don't remove original logic completely. This logic should be fallback when config *enables* spawn of mobs.